go.mod

```diff
@@ -55,4 +55,5 @@ require (
 	k8s.io/client-go v0.16.6
 	k8s.io/code-generator v0.16.6
 	k8s.io/klog v1.0.0
+	k8s.io/metrics v0.16.6
```

Comment:
this version is tight to the client-go version which is `v0.16.6`, to update it to latest version we need to update all the following related library too:
```
        k8s.io/api v0.16.6
	k8s.io/apimachinery v0.16.6
	k8s.io/client-go v0.16.6
	k8s.io/code-generator v0.16.6
```
Also codes needs to be updated too to use these updated packages, not sure how much though. That's why i didn't use the latest version.
I think it should be done in a seperate PR, because it could be a major change.